main/network.c

```diff
@@ -512,9 +512,11 @@ PHPAPI int php_network_parse_network_address_with_port(const char *addr, zend_lo
 	zend_string *errstr = NULL;
 #if HAVE_IPV6
 	struct sockaddr_in6 *in6 = (struct sockaddr_in6*)sa;
-#endif
 
+	memset(in6, 0, sizeof(struct sockaddr_in6));
+#else
 	memset(sa, 0, sizeof(struct sockaddr));
```

Comment:
```suggestion
	memset(in4, 0, sizeof(struct sockaddr_in));
```

to keep this consistent. struct sockaddr could theoretically be smaller than sockaddr_in.